contracts/interfaces/ISablier.sol

```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: Apache-2.0
+// SPDX-License-Identifier: MIT
```

Comment:
same license for external lib or not ?

Reply:
right now we will not touch the ext libs